CONTRIBUTING.md

```diff
@@ -160,174 +204,115 @@ information treat the first 80 characters as a title. Even Github itself does
 
-The code style is enforced by [prettier](https://prettier.io/) which means that in most of the cases you don't actually need to do anything more than running the appropriate task.
-
-To fix any fixable codestyle issue in either SDK or Wallet, you may just run the following command on the respective folder:
-
-```bash
-npm run lint
+We maintain two separate change logs, one for the SDK and another one for the dApp. They can be found at:
+```
+raiden-ts/CHANGELOG.md
+raiden-dapp/CHANGELOG.md
 ```
 
-Linting plugins available for modern IDEs are also useful in early spotting any mistakes and help keep the code quality level.
-
-##### HTML formating
+Please categorize your entry according to the type of your change. The most commonly used ones are `Added`, `Fixed`.
 
-There are some scenarios where prettier will break a template in a weird way:
+👎 Instead of committing for a bug fix:
 
-```html
-<template
-    ><section>
-        <div>
-        <div>
-            <div><p>paragraph</p></div>
-        </div>
-        </div>
-    </section></template
-    >
+```markdown
+### Added
+- A bug fix
 ```
 
-In this case you should go and add line breaks so that the template appears in proper way.
+👍 Do this:
 
-```html
-<template>
-    <section>
-        <div>
-            <div>
-                <div><p>paragraph</p></div>
-            </div>
-        </div>
-    </section>
-</template>
-```
+```markdown
+### Fixed
+- [#777] Specific behavior
 
-##### Vue bindings
+...
 
-For consistency reason the [shorthand](https://vuejs.org/v2/guide/syntax.html#Shorthands) vue bindings are used all across the wallet instead of the full syntax.
+[#777]: https://github.com/raiden-network/light-client/issues/777
+```
 
-```html
-<!-- full syntax -->
-<a v-bind:href="url"> ... </a>
+### Opening a pull request
 
-<!-- shorthand -->
-<a :href="url"> ... </a>
-```
+In order for a Pull Request to get merged into the main repository you should
+have one approved review from one of the core developers of Raiden. Additionally, all
+continuous integration tasks should pass and the build should be green.
 
-```html
-<!-- full syntax -->
-<a v-on:click="doSomething"> ... </a>
+You also need to sign the raiden project CLA (Contributor License
+Agreement). Our CLA bot will help you with that after you created a pull
+request. If you or your employer do not hold the whole copyright of the
+authorship submitted we can not accept your contribution.
 
-<!-- shorthand -->
-<a @click="doSomething"> ... </a>
-```
+For frequent contributors with write access to the repository we have a set of rules on pull requests to signal to our colleagues what their current state is:
 
-```html
-<!-- full syntax -->
-<template v-slot:append> ... </template>
+- **Ready for review**: Means that you consider your work done and it is currently waiting for a review
+- **Draft**: Is considered work in progress and it is not yet ready for a review
```

Comment:
Maybe remove, since we never adhere this

Reply:
We don't even have those labels do we? 🤓

Reply:
@taleldayekh we removed the labels and instead rely on the Draft and Ready for review feature of the Github Pull Requests

Reply:
Ah, I guess you set them when opening the PR.

Reply:
@nephix Can we clarify here that these are not labels, but PR status in the Github PR interface. I actually adhere to this and @andrevmatos is also doing the same from what I remember.

The regular flow should be, I open a draft PR and when I am done I mark it ready for review.

Reply:
From my experience we don't do this at all, or it's at least inconsistent.
I mostly do draft/ready for review, but I often see pull requests in ready for review with `[wip]` or things like these. 
I also very often receive feedback on pull requests that are in draft state.
Because of that, I asked if we should remove it or if we should talk about and align on this.

Reply:
Before removing maybe we should definitely align and discuss them. However, since discussions take time maybe it is better to clarify the original intent and keep it for now, but also add a point for discussion after everyone will be back.